service/tablet_allocator.cc

```diff
@@ -710,12 +710,12 @@ class load_balancer {
             plan.merge(std::move(dc_plan));
         }
 
-        // Make plans for repair jobs
-        plan.set_repair_plan(co_await make_repair_plan(plan));
-
         // Merge table-wide resize decisions, may emit new decisions, revoke or finalize ongoing ones.
         plan.merge_resize_plan(co_await make_resize_plan(plan));
 
+        // Make plans for repair jobs
+        plan.set_repair_plan(co_await make_repair_plan(plan));
```

Comment:
If we rely on order for correct behavior, we better add a comment for someone in the future.